src/Core/Scheduler/Providers/Raster_Provider.js

```diff
@@ -73,81 +51,64 @@ export default {
             throw new Error('layer.url is required');
         }
 
-        const extention = getExtension(layer.url).toLowerCase();
-        const format = `vector/${extention}`;
         layer.options = layer.options || {};
+        // KML and GPX specifications all says that they should be in
+        // EPSG:4326. We still support reprojection for them through this
+        // configuration option
+        layer.projection = layer.projection || 'EPSG:4326';
```

Comment:
Isn't it `reprojection` ?

Reply:
It is the original crs of the layer (see line 61, I use it as crsIn after). I removed the reprojection options, because it is always the parent layer projection, AFAIK.

Reply:
Ok, I was misguided by the comment above